mautic/mautic
# Company leads

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | pending
| Related developer documentation PR URL | n/a
| Issues addressed (#s or URLs) | 
| BC breaks? | n
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR requires previous PR #2403 It lets a contact be assign to a company.

#### Steps to test this PR:
1. Apply PR #2403 and this one
2. on the list of contacts you can now batch assign leads to a company.
3. on a contact's detail view you can now see the companies it belongs to.
4. on a contact's detail view you can add or remove a company
5. on the company's list view you can see the number of leads in each company
6. on the list of contacts you can now filter by company_name:nameOfCompany
